scripts/tools/eslint-plugin-prettier-internal-rules/no-node-comments.js

```diff
@@ -45,7 +46,8 @@ module.exports = {
         }
         const { file, functions } = option;
         return [
-          path.join(__dirname, "../../..", file),
+          // (...)
```

Comment:
I added this comment to avoid reformatting this entire return statement ... argh!